src/java/org/apache/cassandra/index/sai/IndexContext.java

```diff
@@ -692,6 +693,16 @@ public ConcurrentMap<Memtable, MemtableIndex> getLiveMemtables()
         return liveMemtables;
     }
 
+    public @Nullable MemtableIndex getMemtableIndex(Memtable memtable)
+    {
+        return liveMemtables.get(memtable);
+    }
+
+    public @Nullable SSTableIndex getSSTableIndex(Descriptor descriptor)
+    {
+        return getView().getSSTableIndex(descriptor);
```

Comment:
Since `getView()` is an atomic reference, should we consider getting it just once?

Reply:
Looks like this is common practice in the code base, though it seems like we become subject to updates to the underlying view, which could lead to issues.